app/src/main/res/values/strings.xml

```diff
@@ -844,7 +844,7 @@
     <string name="wallet_connect_message_vote">Message: Vote</string>
     <string name="wallet_connect_message_content">Message content</string>
     <string name="wallet_connect_message_reject">Reject</string>
-    <string name="wallet_connect_message_sign_and_return">Sign &amp; Return</string>
+    <string name="wallet_connect_message_sign_and_return">Sign</string>
```

Comment:
This removes Return and doesn’t adjust size.

From what I can read from the bug expected behaviour is the button text size is resized.